### DISCUSSION TITLE
Wrong version logged during start

### DISCUSSION THREAD

**Discussion by Schadows** [dani-garcia/vaultwarden#5315] [Q&A]
### Vaultwarden Support String

After downloading the image from dockerhub with the "latest" tag (which I have confirmed to have the same sha256 id as the v1.32.7 at the time i'm writing this) and starting the docker container, the version present in the starting logs is v1.32.5.
While minor, it is confusing when you think you should be on the latest version and the logs tell you otherwise.

### Vaultwarden Build Version

V1.32.7

### Deployment method

Official Container Image

### Reverse Proxy

no

### Host/Server Operating System

Linux

### Clients

CLI

### Steps To Reproduce

Start the container using the image with the "latest" tag from docker after confirming it shares the same sha256 id as the v1.32.7 version.

### Expected Result

Seeing the version logged during container start be 1.32.7

### Actual Result

The version logged is 1.32.5

### Logs

```text
2024/12/20 14:34:30	stdout	[2024-12-20 13:34:30.622][start][INFO] Rocket has launched from http://0.0.0.0:80
2024/12/20 14:34:30	stdout	
2024/12/20 14:34:30	stdout	\--------------------------------------------------------------------/
2024/12/20 14:34:30	stdout	|   https://github.com/dani-garcia/vaultwarden/issues/new            |
2024/12/20 14:34:30	stdout	| Report suspected bugs/issues in the software itself at:            |
2024/12/20 14:34:30	stdout	|   https://vaultwarden.discourse.group/                             |
2024/12/20 14:34:30	stdout	|   https://github.com/dani-garcia/vaultwarden/discussions or        |
2024/12/20 14:34:30	stdout	| Send usage/configuration questions or feature requests to:         |
2024/12/20 14:34:30	stdout	| official channels to report bugs/features, regardless of client.   |
2024/12/20 14:34:30	stdout	| This is an *unofficial* Bitwarden implementation, DO NOT use the   |
2024/12/20 14:34:30	stdout	|--------------------------------------------------------------------|
2024/12/20 14:34:30	stdout	|                           Version 1.32.5                           |
2024/12/20 14:34:30	stdout	|                        Starting Vaultwarden                        |
2024/12/20 14:34:30	stdout	/--------------------------------------------------------------------\
```

**Comment by BlackDex** [comment]
You should update your container, as it works just fine.
Follow the directions described at the wiki.
